### DISCUSSION TITLE
Missing orientdb-etl-3.2.5.jar and orientdb-teleporter-3.2.5.jar

### DISCUSSION THREAD

**Discussion by rkravie** [orientechnologies/orientdb#9766] [Q&A] [answered]
In the 3.2.5 release orientdb-etl-3.2.5.jar and orientdb-teleporter-3.2.5.jar are not in the plugins directory and oteleporter.sh and oteleporter.bat are not in the bin directory. Were they just forgotten or deliberately left out?

Thanks

**Comment by tglman** [comment, marked as answer]
Hi,

thanks to report this, we are doing some changes in the release process, and this seems to have slipped out, you can find the specific jar on maven central here : https://search.maven.org/artifact/com.orientechnologies/orientdb-teleporter and here https://search.maven.org/artifact/com.orientechnologies/orientdb-etl will try to re-include this from future releases.

Regards
